pkgs/by-name/op/openpam/package.nix

```
@@ -0,0 +1,31 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  autoreconfHook,
+  pkg-config,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "openpam";
+  version = "20230627";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/openpam/openpam/Ximenia/${finalAttrs.pname}-${finalAttrs.version}.tar.gz";
```

```suggestion
    url = "mirror://sourceforge/openpam/openpam/Ximenia/openpam-${finalAttrs.version}.tar.gz";
```

when appending anything like headless or so, that would be wrong

---

That was in the original derivation, but I’ve fixed it to hardcode the name.